core/src/main/scala/kafka/admin/AdminUtils.scala

```diff
@@ -65,6 +67,8 @@ object AdminUtils extends Logging with AdminUtilities {
   val rand = new Random
   val AdminClientId = "__admin_client"
   val EntityConfigChangeZnodePrefix = "config_change_"
+  val objectMapper = new ObjectMapper()
```

Comment:
We should add the relevant methods to `Json` that already has an `ObjectMapper` as a field.